### PR TITLE
for Bratashchuk

### PR DESCRIPTION
**Pull request by MayerLansky**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,8 +5,323 @@
...
+          <li class="links">
+            <a href="#samsung" class="links__link_a">Samsung</a>
+          </li>
+
+          <li class="links">
+            <a href="#smartphones" class="links__link_a">Smartphones</a>
+          </li>
+
+          <li class="links" data-qa="nav-hover">
+            <a href="#laptops" class="links__link_a">Laptops & Computers</a>
```
Comment (OleziO) [inline src/index.html:37]: Divide by a few lines.

File: src/index.html [hunk 2]
```diff
@@ -5,8 +5,323 @@
...
+    </header>
+    
+    <main class="catalog">
+      <div class="card" data-qa="card">
+        <img 
+        src="./images/imac.jpeg" 
+        class="card__img" 
+        alt="APPLE A1419 iMac 27"
+      >
+        <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
```
Comment (OleziO) [inline src/index.html:66]: Divide by a few lines. Check other similar cases.

File: src/index.html [hunk 3]
```diff
@@ -5,8 +5,323 @@
...
+
+        <div class="card__price">
+          <p class="card__price-label">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+
+        <a 
+        href="#buy" 
+        class="card__buy-link" 
+        data-qa="card-hover"
```
Comment (OleziO) [inline src/index.html:87]: Add 2 spaces before.

File: src/styles/main.scss [hunk 4]
```diff
@@ -1,3 +1,239 @@
...
+  position: relative;
+  list-style: none;
+  justify-content: center;
+  text-align: center;
+  display: flex;
+
+  &__link_a {
+    text-decoration: none;
+    text-transform: uppercase;
+    color: #060b35;
```
Comment (OleziO) [inline src/styles/main.scss:62]: Move to the variable.

File: src/styles/main.scss [hunk 5]
```diff
@@ -1,3 +1,239 @@
...
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  background: #00acdc;
+  border-radius: 8px;
+}
+
+.catalog {//add media ??
+  display: flex;
+  flex-direction: row;  //????????
```
Comment (OleziO) [inline src/styles/main.scss:102]: You don't need to use this, cause the row is the default value.

File: src/styles/main.scss [hunk 6]
```diff
@@ -1,3 +1,239 @@
...
+  justify-content: center; //???sb
+  flex-wrap: wrap;
+  // box-sizing: border-box;
+  padding: $padding-vertical $padding-horizont;
+  // padding: 50px;
```
Comment (OleziO) [inline src/styles/main.scss:107]: Remove all comments from the code.

File: src/styles/main.scss [hunk 7]
```diff
@@ -1,3 +1,239 @@
...
+  height: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:first-child,
```
Comment (OleziO) [inline src/styles/main.scss:233]: Replace with -n + 1 to make the code the same style.

**Review by OleziO — CHANGES_REQUESTED** [review]
Add demo link to PR description.

**Comment by MayerLansky** [comment]
[DEMO LINK](https://MayerLansky.github.io/layout_catalog/)
[TEST REPORT LINK](https://MayerLansky.github.io/layout_catalog/report/html_report/)

**Comment by MayerLansky** [comment]
[DEMO LINK](https://MayerLansky.github.io/layout_catalog/)
[TEST REPORT LINK](https://MayerLansky.github.io/layout_catalog/report/html_report/)

**Pull request closed without merging**